packages/react-lowcode/src/codegen/ast/widgetDeclaration.ts

```diff
@@ -100,4 +126,43 @@ function isTableDeclaration(node: ts.Node): boolean{
 function isDetailDeclaration(node: ts.Node): boolean{
     ///TODO: check also for export key word
     return ts.isVariableDeclaration(node)
+}
+
+function isElementWithAttributes(node: ts.Node): node is ts.JsxOpeningElement | ts.JsxSelfClosingElement {
+    return (ts.isJsxOpeningElement(node) || ts.isJsxSelfClosingElement(node)) && node.attributes.properties.length > 0;
+}
+
+function getAttributeValue(attribute: ts.JsxAttribute): string | undefined {
+    let result: string | undefined;
+
+    if (attribute.initializer) {
+        result = getStringTypeAttributeValue(attribute.initializer)
+            || getNumberTypeAttributeValue(attribute.initializer)
+            || getBooleanTypeAttributeValue(attribute.initializer);
+    } else {
+        result = 'true';
+    }
+
+    return result;
+}
+
+function getStringTypeAttributeValue(initializer: ts.StringLiteral | ts.JsxExpression): string | undefined {
+    return ts.isStringLiteral(initializer) ? initializer.text : undefined;
+}
+
+function getNumberTypeAttributeValue(initializer: ts.StringLiteral | ts.JsxExpression): string | undefined {
+    if (ts.isJsxExpression(initializer) && initializer.expression !== undefined && ts.isNumericLiteral(initializer.expression)) {
+        return initializer.expression.getText();
+    } else {
+        return undefined;
```

Comment:
no need to return undefined (it is implicit) - please modify also the other code

Reply:
ok, it's fixed